packages/trackerless-network/test/integration/Inspect.test.ts

```diff
@@ -30,7 +30,8 @@ describe('inspect', () => {
             layer0: {
                 entryPoints: [publisherDescriptor],
                 peerDescriptor,
-                transport
+                transport,
+                stopGivenTransport: true
```

Comment:
Maybe we could use the same pattern as in `dht` test utils:
https://github.com/streamr-dev/network/blob/e88eacad4010d26b76583b0339d2a3dbb8f9e80e/packages/dht/test/utils/utils.ts#L75-L76

Or we could add e.g. `stop` event which the creator of the instance could listen.